### ISSUE TITLE
Publish to addons.mozilla.org

### ISSUE THREAD

**Issue by strugee** [panicsteve/cloud-to-butt#65]
According to https://www.extensiontest.com/test/01101f90-a993-11e7-9f5a-4de5e2dedc5e this extension is actually compatible with Firefox out-of-the-box. It'd be nice if it was submitted to addons.mozilla.org for people to install. DaveRandom/cloud-to-butt-mozilla could then be folded into this project.

**Comment by andreicristianpetcu** [comment]
[Done](https://addons.mozilla.org/en-US/firefox/addon/cloud-2-butt-plus/)

It took me like [3 minutes](https://vimeo.com/241435808) :D

**Comment by strugee** [comment]
😍

**Comment by strugee** [comment]
I wonder if @panicsteve wants write access though...?

**Comment by andreicristianpetcu** [comment]
If he gave me his AMO account, I would.

**Comment by panicsteve** [comment]
Hi Andrei,

I created a profile as panicsteve: 

https://addons.mozilla.org/en-US/firefox/user/panicsteve/

**Comment by andreicristianpetcu** [comment]
I think I made you an owner :) I added you by your email stevenf@panic.com.

You should see the admin page https://addons.mozilla.org/en-us/developers/addon/cloud-2-butt-plus/edit

**Comment by andreicristianpetcu** [comment]
No, thank YOU for getting rid of the horrors of "Cloud Computing" and for making the Internet a better place :)
